lib/iris/fileformats/netcdf/saver.py

```diff
@@ -1864,8 +1864,20 @@ def _create_cf_grid_mapping(self, cube, cf_var_cube):
         None
 
         """
-        cs = cube.coord_system("CoordSystem")
-        if cs is not None:
+        # TODO(CB): We need ALL coord systems here, not just the fist one found.
+        # cs = cube.coord_system("CoordSystem")
+
+        # TODO(CB): This is workaround code (gets _unique_ coord systems):
+        # TODO(CB): Put this code in new cube.coord_systems() function, or modify cube.coord_system()
+        # coord_systems = [coord.coord_system for coord in cube.coords() if coord.coord_system]
+        coord_systems = []
+        for coord in cube.coords():
+            if coord.coord_system and coord.coord_system not in coord_systems:
+                coord_systems.append(coord.coord_system)
```

Comment:
If this code sticks around, I think you'd be better off with a `set` (instead of a `list`).